pkgs/stdenv/generic/check-meta.nix

```diff
@@ -329,16 +319,13 @@ let
     unfree = bool;
     unsupported = bool;
     insecure = bool;
-    # TODO: refactor once something like Profpatsch's types-simple will land
-    # This is currently dead code due to https://github.com/NixOS/nix/issues/2532
-    tests = attrsOf (mkOptionType {
+    tests = {
       name = "test";
-      check = x: x == {} || ( # Accept {} for tests that are unsupported
+      verify = x: x == {} || ( # Accept {} for tests that are unsupported
         isDerivation x &&
         x ? meta.timeout
       );
-      merge = lib.options.mergeOneOption;
-    });
+    };
```

Comment:
Is this correct? I'm struggling to find a `meta.tests`. `passthru.tests` is the convention.

Reply:
IIRC `tests` is a normal package attribute, and not in `meta` because it would make `meta` non-serializable.